.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "eslint-config-ndla-h5p",
+    "overrides": [
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
```

Comment:
This should be `2021` in your case, IMHO. It makes more sense not to treat global variables and syntax differently.